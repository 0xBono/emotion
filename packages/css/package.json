{
  "name": "@emotion/css",
  "version": "0.9.8",
  "description": "a function to serialize css and object styless",
  "main": "dist/css.cjs.js",
  "module": "dist/css.esm.js",
  "license": "MIT",
  "repository": "https://github.com/emotion-js/emotion/tree/master/next-packages/css",
  "dependencies": {
    "@emotion/serialize": "^0.9.1",
    "@emotion/utils": "^0.8.2",
    "babel-plugin-emotion": "^9.2.10"
  },
  "publishConfig": {
    "access": "public"
  },
  "files": [
    "src",
    "dist",
    "macro.js"
  ],
  "umd:main": "./dist/index.min.js",
  "browser": {
    "./dist/css.cjs.js": "./dist/css.browser.cjs.js",
    "./dist/css.esm.js": "./dist/css.browser.esm.js"
  }
}
