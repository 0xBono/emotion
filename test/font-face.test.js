/* eslint-disable jsx-quotes,no-useless-escape,no-template-curly-in-string */
/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'
import { matcher, serializer } from '../jest-utils'

// eslint-disable-next-line no-unused-vars
import css, { sheet, fragment, fontFace } from '../src/index'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

// This does nothing
describe.skip('font-face', () => {
  test('adds font-face to sheet', done => {
    fontFace`
      font-family: 'Patrick Hand SC';
      font-style: normal;
      font-weight: 400;
      src: local('Patrick Hand SC'), local('PatrickHandSC-Regular'), url(https://fonts.gstatic.com/s/patrickhandsc/v4/OYFWCgfCR-7uHIovjUZXsZ71Uis0Qeb9Gqo8IZV7ckE.woff2) format('woff2');
      unicode-range: U+0100-024F, U+1E00-1EFF, U+20A0-20AB, U+20AD-20CF, U+2C60-2C7F, U+A720-A7FF;
    `
    // console.log(sheet.rules())
    expect(sheet).toMatchSnapshot()
  })
})
