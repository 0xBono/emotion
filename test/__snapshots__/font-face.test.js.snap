// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`font-face adds font-face to sheet 1`] = `Array []`;
